src/igz_controls/components/default-dropdown/default-dropdown.component.js

```diff
@@ -399,7 +399,7 @@
 
                 ctrl.isDropdownContainerShown = false;
             }
-            event.stopPropagation();
+            event && event.stopPropagation();
```

Comment:
I don't like using a non-boolean expression as a boolean.
I'd like to use both operands of `&&` as booleans.
I suggest something like this:
```js
if (!lodash.isNil(event)) {
    event.stopPropagation();
}
```
Or:
```js
lodash.result(event, 'stopPropagation');
```